docs/metric_deletion_and_retention.md

```diff
@@ -0,0 +1,133 @@
+# Deleting Data
+
+Promscale supports series deletion & metric retention. Series deletion deletes all data for a specified series over the entire time-range.
+Metric retention deletes data older than a specified duration for a given metric. You can specify a default metric retention period as well as overwrite that default on a particular metric.
+
+Deletion of data in Promscale can be categorized under:
+
+1. Deletion of series (API and SQL **(todo)**)
+2. Deletion of metric
+3. Deletion of series/metric by time (API **(todo)** and SQL)
+4. Metric retention (Deletion via scheduling data retention policies (SQL))
+
+## Deletion of series (API)
+
+Promscale exposes an API for deletion of series, simply by using the label_matchers. This works same as prometheus delete API.
+However, for deleting data in Promscale, you will need to enable permissions for advanced users. This is done by setting
+[`web-enable-admin-api`](https://github.com/timescale/promscale/blob/master/docs/cli.md#general-flags) flag to `true`.
+
+**Note**: The start and end timestamp options are not supported yet. Currently, the `delete_series` API lets you delete the series across the entire time-range only.
+
+#### API
+
+URL query parameters:
+
+* **match[]=<series_selector>**: Repeated label matcher argument that selects the series to delete. At least one match[] argument must be provided.
+
+```
+POST /delete_series
+PUT /delete_series
+```
+
+**Example:**
+
+For deleting series that have labels as `job="prometheus"` and a regex label `instance="prom.*"`.
+
+```shell
+curl -X POST -g http://localhost:9201/delete_series?match[]={job="prometheus", instance=~"prom.*"}
+```
+
+## Deletion of metric
+
+Promscale allows you to delete an entire metric both via SQL and API.
+
+### SQL
+
+You can delete an entire metric from the database using the function `prom_api.drop_metric(metric_name_to_be_dropped text)`.
```

Comment:
we should note that this can only be done when no promscale instances are connected to the db as this is an administrative command